app/Http/Controllers/Api/Application/Nodes/AllocationController.php

```diff
@@ -93,4 +95,36 @@ public function delete(DeleteAllocationRequest $request, Node $node, Allocation
 
         return new JsonResponse([], JsonResponse::HTTP_NO_CONTENT);
     }
+
+    /**
+     * Deletes multiple allocations from the Panel.
+     *
+     * @throws \Pterodactyl\Exceptions\Service\Allocation\ServerUsingAllocationException
+     */
+    public function delete_multiple(DeleteAllocationsRequest $request, Node $node): JsonResponse
+    {
+        $data = $request->validated();
+
+        //Generates an array of allocations that the user specified that aren't real allocations
+        $node_ids = array_map(function($allocation) {
+            return $allocation['id'];
+        }, $node->allocations()->get()->toArray());
+
+        //If there are any allocations that do not exist, throw an error
+        if(count(array_diff($data['ids'], $node_ids))>0)
+        {
```

Comment:
Please format your code properly, we have a PHP CS Fixer file for a reason.  You should be able to just re-format all your code using that tool.